.github/workflows/main.yml

```
@@ -0,0 +1,51 @@
+name: CI
+
+on: push
+
+jobs:
+  confirm_config_and_documentation:
+    runs-on: ubuntu-latest
+    name: Confirm config and documentation
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+          bundler-cache: true
+      - run: bundle exec rake confirm_config documentation_syntax_check confirm_documentation
+
+  main:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby:
+          - 2.4
+          - 2.5
+          - 2.6
+          - 2.7
+          - 3.0
```

```suggestion
          - "3.0"
```
Don't let YAML be confused about `3` and `3.0`.